scripts/flexible-server/common-functions.sh

```diff
@@ -19,12 +19,21 @@ function get_flexible_sql_servers() {
     area_selector="| where tags.businessArea == '$2'"
   fi
 
+  if [ -z $3 ]; then
```

Comment:
Looks good. Is the `''` there for servers that don't have a replica? Curious cos otherwise it seems like only 3 arguments should be necessary otherwise

Reply:
It's because the status script doesn't care about replicas so we want all servers returned in that case. I found I had to include the empty string in the auto start as it was interpreting the argument for the wrong thing e.g. setting businessArea to 'replica'

If you can spot an easy fix lmk cause it was wrecking my head trying to find one lol

Reply:
That do be the problem with using positional parameters. Maybe worth a tech improvement to do to this repo what Marty did to https://github.com/hmcts/dtspo-daily-monitoring from S&T a couple of weeks ago

Reply:
Good shout I reckon!